docs/src/components/PlaygroundSettingsShaders.tsx

```diff
@@ -0,0 +1,87 @@
+import toast from 'react-hot-toast';
+import { Tooltip } from 'react-tooltip';
+import styles from './PlaygroundSettingsShaders.module.css';
+
+export default function PlaygroundSettingsShaders() {
+  return (
+    <div className={styles.container}>
+      <div className={styles.headerContainer}>
+        <div className={styles.headerInputLabel}>Image ID</div>
+        <div className={styles.headerDescriptionLabel}>Description</div>
+      </div>
+      <ShaderInfo
+        shader_id="remove_greenscreen"
+        description="Shader removing green color."
```

Comment:
```suggestion
        description="Shader removing a green background. It supports exactly one child component."
```
btw this could paramterized so user can specify color to remove(not in this PR, but maybe followup)